### ISSUE TITLE
Compatibility with Nest 8

### ISSUE THREAD

**Issue by PedroNunezNexplore** [ferrerojosh/nest-keycloak-connect#72]
Hi @ferrerojosh, and contributors! Great library!! Congrats

I write you because I'm having troubles to use the library with the new Nest libraries version (8.0.4). This is the error trying to install `nest-keycloak-connect` on a new Nest project:

```
npm ERR! code ERESOLVE
npm ERR! ERESOLVE unable to resolve dependency tree
npm ERR! 
npm ERR! While resolving: pruebanestkeycloak@0.0.1
npm ERR! Found: @nestjs/common@8.0.4
npm ERR! node_modules/@nestjs/common
npm ERR!   @nestjs/common@"^8.0.0" from the root project
npm ERR! 
npm ERR! Could not resolve dependency:
npm ERR! peer @nestjs/common@"^7.0.3" from nest-keycloak-connect@1.6.1
npm ERR! node_modules/nest-keycloak-connect
npm ERR!   nest-keycloak-connect@"*" from the root project
```

Is it my fault, or is there a way to solve it?
If not, is there a plan to update `nest-keycloak-connect` to solve the issue?

Thanks!
Best Regards

**Comment by ferrerojosh** [comment]
This will come as a fix with #69, setting nest as a peerDependency

**Comment by PedroNunezNexplore** [comment]
Thanks @ferrerojosh !! great news
